mha_helper/ssh_helper.py

```diff
@@ -61,7 +61,7 @@ def make_ssh_connection(self):
         parser.add_option('-o', '--additional_options', action='append', type='string')
         parser.add_option('-i', '--key_file_path', type='string')
 
-        (options, args) = parser.parse_args(shlex.split(self._ssh_options))
+        (options, args) = parser.parse_args(shlex.split(self._ssh_options) if self._ssh_options else [])
```

Comment:
By default, [shlex.split()](https://docs.python.org/2/library/shlex.html) reads from stdin when None is passed in.